src/App.js

```diff
@@ -1,12 +1,75 @@
-import logo from './logo.svg';
 import './App.css';
+import {useEffect, useState} from 'react';
+import './disign.css';
+import ObjectInfo from "./components/ObjectInfo";
 
 function App() {
-  return (
-    <div className="App">
-        <div> hello </div>
+
+    const [leftarr, setlarr] = useState([]);
+    const [rightarr, setrarr] = useState([]);
+    const [ backgroundColor, setBackgroundColor ] = useState('#50e3e3');
+
+    useEffect(() => {
+        const dataJSON = require('./MOCK_DATA.json');
+        setlarr(dataJSON);
+        console.log(leftarr.length);
+    }, [])
+
+    useEffect(() => {
+        if(rightarr.length >= 10){
+            setBackgroundColor('#db6fe8');
+        }else{
+            setBackgroundColor('#50e3e3');
+        }
+
+    }, [rightarr])
+
+    function onItemPress({side, id, gender}) {
+        if (side === 'Left') {
+            if (conditionCheck(gender)) {
+                const from = {arr: leftarr, setArr: setlarr};
+                const to = {arr: rightarr, setArr: setrarr};
+                moveElement({id, from, to});
+            } else {
+                alert('Max 5 male and 5 female on right side');
+            }
+        } else {
+            const from = {arr: rightarr, setArr: setrarr};
+            const to = {arr: leftarr, setArr: setlarr};
+            moveElement({id, from, to});
+        }
+    }
+
+    function conditionCheck(gender) {
+        let checkState = true;
+        if (gender === 'Male' || gender === 'Female') {
+            const checkArr = rightarr.filter(item => item.gender === gender);
+            if(checkArr.length===5){checkState = false}
+        }
+        return checkState
```

Comment:
```suggestion
        if (gender === 'Male' || gender === 'Female') {
            const checkArr = rightarr.filter(item => item.gender === gender);
           return checkArr.length === 5;
        }
        return true
```

checkArr.length === 5; already returns boolean